e2e_tests/src/e2e_conduit/scenarios/filter_scenario.py

```diff
@@ -0,0 +1,40 @@
+import e2e_conduit.fixtures.importers as importers
+import e2e_conduit.fixtures.processors as processors
+import e2e_conduit.fixtures.exporters as exporters
+from e2e_conduit.scenarios import Scenario, scenarios
+
+
+def app_filter_indexer_scenario(sourcenet):
+    return Scenario(
+        "app_filter_indexer_scenario",
+        importer=importers.FollowerAlgodImporter(sourcenet),
+        processors=[
+            processors.FilterProcessor([
+                {"any": [
+                    {"tag": "txn.type",
+                     "expression-type": "equal",
+                     "expression": "appl"
+                    }
+                ]}
+            ]),
+        ],
+        exporter=exporters.PostgresqlExporter(),
+    )
+
+
+def pay_filter_indexer_scenario(sourcenet):
+    return Scenario(
+        "pay_filter_indexer_scenario",
+        importer=importers.FollowerAlgodImporter(sourcenet),
+        processors=[
+            processors.FilterProcessor([
+                {"any": [
+                    {"tag": "txn.type",
+                     "expression-type": "equal",
+                     "expression": "pay"
+                    }
+                ]}
+            ]),
+        ],
+        exporter=exporters.PostgresqlExporter(),
+    )
```

Comment:
AlgodImporter would also work for these filter tests, right?

Reply:
The `AlgodImporter` fixture is separate still since the plugins were originally two separate ones instead of one w/ a config option. In this case we still need to use the follower because the pgsql exporter will fail w/o the deltas.